plugins/caddy/configuration.go

```diff
@@ -388,8 +391,9 @@ func parseConfiguration(cfg *Configuration, h *caddyfile.Dispenser, isBlocking b
 					case "api_key":
 						cdn.APIKey = h.RemainingArgs()[0]
 					case "dynamic":
-						if len(h.RemainingArgs()) > 0 {
-							cdn.Dynamic, _ = strconv.ParseBool(h.RemainingArgs()[0])
+						args := h.RemainingArgs()
+						if len(args) > 0 {
+							cdn.Dynamic, _ = strconv.ParseBool(args[0])
```

Comment:
```suggestion
						cdn.Dynamic = true
						args := h.RemainingArgs()
						if len(args) > 0 {
							cdn.Dynamic, _ = strconv.ParseBool(args[0])
```
To support users that declare booleans directives with only the directive in the Caddyfile

e.g.
```
{
    cache {
        cdn {
            dynamic
        }
    }
}
```

Reply:
Ah yes! I notice this was possible too. Let me change that…